src/JsonSchema/Uri/UriRetriever.php

```diff
@@ -40,19 +40,19 @@ class UriRetriever implements BaseUriRetrieverInterface
      *
      * @param UriRetrieverInterface $uriRetriever
      * @param string $uri
-     * @return bool|void
+     * @return bool
      */
     public function confirmMediaType($uriRetriever, $uri)
     {
         $contentType = $uriRetriever->getContentType();
 
         if (is_null($contentType)) {
             // Well, we didn't get an invalid one
-            return;
+            return true;
```

Comment:
I'm not sure, how many users uses this method, but changing from null to a boolean value could be problematic, especially in code something like:

```
if ( confirmMediaType( $uriRetriever, $uri ) ) {
  // the code that runs when true
} else {
  // the former "null" value-code
}
```
It now would break, as null evaluates to false, and not to true. So in some cases, this could be a problem. However, the new values makes sense, too. I'm really not sure :(